.gitignore

```diff
@@ -4,4 +4,12 @@ yarn.lock
 package-lock.json
 cireports
 package.json
-gulpfile.js
\ No newline at end of file
+gulpfile.js
```

Comment:
Whether these changes needed?

Reply:
I have completed all the above mentioned changes sir